formattest/testdata/indent_line_comments/input.yaml

```diff
@@ -0,0 +1 @@
+a: 1 # comment
```

Comment:
Something I was gonna try on my PR but didn't get to is adding a head and foot comment and ensure that those aren't affected by any line comment specific logic. Would you be able to add that?